### ISSUE TITLE
Feature request: ability to set datetime

### ISSUE THREAD

**Issue by gjdv** [arska/python-daikinapi#16]
There are some suggestions that one can set the datetime using `notify_date_time` by calling:
```
/common/notify_date_time?lpw=&date=2020/02/01&zone=GMT&time=22:24:00
```
https://github.com/ael-code/daikin-control/issues/26#issuecomment-581078687
https://community.openhab.org/t/daikin-airbase-binding/77179/233

But using that format doesn't seem to work for me. notify_date_time does return a `ret=OK` status, but
calling `/common/get_datetime` before and after a call to `notify_date_time` shows that it had no effect.

Do you (/anyone else) have experience with this and/or able to test it?

As part of pull request #15, I implemented a getter for datetime. Would be nice to add a (working) setter.

**Comment by arska** [comment]
No idea here, keeping this open if somebody else finds a working solution to set the date & time on these devices via API

**Comment by gjdv** [comment]
I got a little wiser. The Android app Daikin Online Controller sets the datetime of the airco units (procedure: get_datetime on the device shows time being offset to the current time; open the Android app, check the settings in there for an airco unit and see that they are updated to current timestamp of android device; perform get_datetime again, and observe that time is indeed updated on the airco unit).
Unfortunately I haven't been able yet to snif the network traffic between the two devices (airco unit and android device), e.g., using WireShark; but with that it would be possible to figure out how the notify_date_time messages should be constructed.
btw... shocking to see that the internal clock drifts for more than 3 hours over a 2 month time period

**Comment by gjdv** [comment]
Found out, using a sniffer, that the Daikin API is quite specific if it comes to the urlencoding of date- and time-separators and does not accept leading zeros. Solved in #17.

**Comment by arska** [comment]
Thanks for sniffing and contributing!
